src/explorer/model/DependenciesMenu.ts

```diff
@@ -54,7 +54,8 @@ export class DependenciesMenu extends Menu implements ITreeItem {
 
     private _savePom(): void {
         const textEditors: vscode.TextEditor[] = vscode.window.visibleTextEditors;
-        const textEditor: vscode.TextEditor | undefined = textEditors.find(editor => editor.document.fileName === this.project.pomPath);
+        const textEditor: vscode.TextEditor | undefined = textEditors.find(editor =>
```

Comment:
` vscode.Uri.file(this.project.pomPath)` might be expensive. It's better to construct it outside.